routers/user/home.go

```diff
@@ -115,6 +129,7 @@ func retrieveFeeds(ctx *context.Context, user *models.User, includePrivate, isPr
 				return
 			}
 		}
+
```

Comment:
Nor did gofmt add it :smile:. I'm generally opposed to unnecessary whitespace changes, particularly when they are far away from a commit's actual changes; they end up making the git history more complex (i.e. looking at a file's blame).